project/CelebornBuild.scala

```diff
@@ -56,6 +56,8 @@ object Dependencies {
   val ratisVersion = "2.5.1"
   val roaringBitmapVersion = "0.9.32"
   val rocksdbJniVersion = "8.5.3"
+  val jacksonVersion = "2.15.3"
+  val paranamerVersion = "2.8"
```

Comment:
seems we don't need to manage `paranamerVersion` explicitly, let's simply delegate it to jackson

Reply:
> seems we don't need to manage `paranamerVersion` explicitly, let's simply delegate it to jackson

Thank you. I remove it.